maskrcnn_benchmark/engine/trainer.py

```diff
@@ -44,10 +45,19 @@ def do_train(
     device,
     checkpoint_period,
     arguments,
+    tb_log_dir,
```

Comment:
What about passing the `meter` as an argument to the `do_train`? This way, we can define it in the `tools/train.py` file.